addons/point_of_sale/static/src/js/Screens/ReceiptScreen/ReceiptScreen.js

```diff
@@ -43,6 +43,12 @@ export class ReceiptScreen extends AbstractReceiptScreen {
                     await this.handleAutoPrint();
                 }
             }, 0);
+
+            // When the order is paid, if there is still a part of the order
+            // to send in preparation it is automatically sent
+            if (this.pos.globalState.orderPreparationCategories.size) {
+                this.pos.sendOrderInPreparation(this.currentOrder);
+            }
```

Comment:
When opening a restaurant session, does it still make sense to call this after payment? What if the order is paid and some items are not submitted, this will submit the full order, right?

Reply:
We talked about this with Dama, in fact, you always have to send the commands to the preparation display when it is activated. 
This function will only return the unsent items taking into account the sending history.

Reply:
Perfect.